tests/rt.conf

```diff
@@ -71,6 +71,9 @@ RUN | cpld_control_c48                                   |
 COMPILE | s2swa_faster | intel | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DFASTER=ON | - cheyenne  noaacloud  | fv3 |
 RUN | cpld_control_p8_faster                            | - cheyenne  noaacloud                 | baseline |
 
+COMPILE | s2sa | intel | -DAPP=S2SA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | - noaacloud | fv3 |
```

Comment:
This test doesn't actually require a separate compile since whether waves are coupled or not is controlled via the namelist. So, for example you can see that the ``cpld_control_nowave_noaero_p8`` uses the ``app=s2sw`` compile. The wave model is compiled, but not included via run-time configuration.

You should be able to add the new test along side the other s2swa tests and that saves one compile job. All you would need is your RUN line after LN 50 (the cpld_bmark_p8 test). 

You could add a comment like
```
RUN | cpld_restart_bmark_p8                             | - s4 jet cheyenne acorn   noaacloud   |          | cpld_bmark_p8

# Aerosol, no Wave
RUN | cpld_s2sa_p8 | - noaacloud | baseline |
....
```